### PR TITLE
Dynamically declare BASE_URL

### PR DESCRIPTION
**Pull request by chalx**
If `BASE_URL` environment variable exists it will be used, else fallback to the one from .env file

### CODE REVIEW

File: simplq/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+BASE_URL="http://localhost:8080"
```
Comment (daltonfury42) [inline simplq/.env:1]: Lets fallback to `https://devbackend.simplq.me/v1`

File: simplq/package.json [hunk 2]
```diff
@@ -36,13 +36,14 @@
     "styled-components": "^5.2.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "npm run config && react-scripts start",
+    "build": "npm run config && react-scripts build",
```
Comment (daltonfury42) [inline simplq/package.json:40]: We are configuring the url before build. Can we configure UI to take up the backend url dynamically after build?

Easiest way I could think of is to generate a json config object in the public folder and vary it from inside the frontend server. Then the client would request for this json object and derive the backend URL at start. 

I don't know if will lead to issues where a backend request happens before the backend URL is resolved.

Comment (chalx) [reply simplq/package.json:40]: @daltonfury42 I don't think it should be so complicated.

**Review by daltonfury42 — APPROVED** [review]
Let's merge this, and optimise later. Thanks

**Pull request merged**